mersenne-31/src/quintic.rs

```diff
@@ -0,0 +1,96 @@
+use core::ops::Mul;
+
+use p3_field::{
+    define_ext, field_to_array, impl_ext_af, impl_ext_f, AbstractExtensionField, AbstractField,
+    Field,
+};
+
+use crate::Mersenne31;
+
+// F[X]/(X^5 - X - 6)
```

Comment:
Was there a reason you picked this polynomial in particular?

Reply:
No, not really. It was the smallest c for X^5-X-c. Definitely worth thinking about harder, will add todo